tests/test_messages.py

```diff
@@ -261,17 +261,19 @@ def test_call_error_representation():
 def test_call_representation():
     call = Call(unique_id="1", action=Action.Heartbeat, payload={})
 
-    assert str(call) == "<Call - unique_id=1, action=Heartbeat, payload={}>"
+    assert str(call) == (
+        f"<Call - unique_id=1, action={Action.Heartbeat}" ", payload={}>"
```

Comment:
* For `python >= 3.11`, `Action.Heartbeat` returns `"Action.Heartbeat"`.
* For `python < 3.11`, `Action.Heartbeat` returns `"Heartbeat"`.